pages/services/edge-lb/1.0.0/installing/index.md

```diff
@@ -85,18 +85,13 @@ Use the following CLI commands to provision the Edge-LB service account with the
 
 All CLI commands can also be executed via the [IAM API](/1.10/security/iam-api/).
```

Comment:
how will this link work once dcos 1.11 comes out? We need to figure out a way where you're not hard coding links to dcos versions.

Reply:
I believe there is a way to add redirects from /XYZ to /version/XYZ